src/main/java/org/commonjava/util/partyline/impl/infinispan/model/FileMeta.java

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc. (nos-devel@redhat.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.commonjava.util.partyline.impl.infinispan.model;
+
+import org.commonjava.util.partyline.lock.LockLevel;
+import org.commonjava.util.partyline.lock.local.LocalLockOwner;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.Externalizable;
+import java.io.IOException;
+import java.io.ObjectInput;
+import java.io.ObjectOutput;
+import java.util.Date;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class FileMeta
+        implements Externalizable
+{
+    private String filePath;
+
+    private boolean directory;
+
+    private Date createdDate;
+
+    private Date lastModifiedDate;
+
+    private FileBlock firstBlock;
+
+    private Map<String, String> metadataMap = new ConcurrentHashMap<>();
+
+    private Map<String, LockLevel> lockMap = new ConcurrentHashMap<>();
+
+    private LocalLockOwner lockOwner;
```

Comment:
We should remove this in favor of using the field in InfinispanJF and modding InfinispanJFS.close() to take the lockOwner as a separate param.